### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Yehorkk**
- [DEMO LINK](https://yehorkk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://yehorkk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,73 @@
...
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+              >
+              Laptops & Computers
+            </a>
+          </li>
```
Comment (xiimiik) [inline src/index.html:49]:
```suggestion
          <li class="nav__item">
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+  --accent-color: #00acdc;
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
```
Comment (xiimiik) [inline src/style.css:11]: make it class

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+}
```
Comment (xiimiik) [inline src/style.css:47]: just add list-style: none to `.nav__list`, and remove for nav__item
```suggestion
.nav__list {
  display: flex;
  padding: 0;
  margin: 0;
}

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,67 @@
...
+
+header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.logo {
+  padding: 10px 0;
+}
```
Comment (xiimiik) [inline src/style.css:19]: u can remove this, center elements in header

**Review by xiimiik — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -6,9 +6,73 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="moyo"
+          class="logo__image">
```
Comment (loralevitska) [inline src/index.html:18]:
```suggestion
          class="logo__image"
        >
```

File: src/index.html [hunk 6]
```diff
@@ -6,9 +6,73 @@
...
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+              >
```
Comment (loralevitska) [inline src/index.html:46]:
```suggestion
            >
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,63 @@
...
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+}
```
Comment (loralevitska) [inline src/style.css:43]: not fixed
image

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,63 @@
...
+  --accent-color: #00acdc;
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
```
Comment (loralevitska) [inline src/style.css:11]: not fixed
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Pull request closed without merging**